sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterImpl.java

```diff
@@ -233,20 +234,14 @@ public boolean enableRBAC() {
                             }));
     }
 
-    @Override
-    public KubernetesClusterImpl withVersion(KubernetesVersion kubernetesVersion) {
-        this.inner().withKubernetesVersion(kubernetesVersion.toString());
-        return this;
-    }
-
     @Override
     public KubernetesClusterImpl withVersion(String kubernetesVersion) {
         this.inner().withKubernetesVersion(kubernetesVersion);
         return this;
     }
 
     @Override
-    public KubernetesClusterImpl withLatestVersion() {
+    public KubernetesClusterImpl withDefaultVersion() {
```

Comment:
default is always latest?

Reply:
No, default is actually not the latest.
![image](https://user-images.githubusercontent.com/15213781/93046411-4d3e3c00-f68c-11ea-9f55-60e352d2949a.png)